### DISCUSSION TITLE
Trojan-go无法正常使用

### DISCUSSION THREAD

**Discussion by Sora39831** [SagerNet/SagerNet#97] [Q&A] [answered]
版本:SN-0.1-rc10-arm64-v8a.apk
错误:未知插件:trojan-go-plugin
![IMG_20210517_135415](https://user-images.githubusercontent.com/36908878/118439176-07081380-b718-11eb-9ad0-358da574fb08.jpg)
![IMG_20210517_135354](https://user-images.githubusercontent.com/36908878/118439182-08d1d700-b718-11eb-9738-5f04f8458927.jpg)
![IMG_20210517_135239](https://user-images.githubusercontent.com/36908878/118439184-0a030400-b718-11eb-8e86-21291dc3e118.jpg)

**Comment by nekohasekai** [comment, marked as answer]
https://t.me/sagernet/30
